mq/main/pom.xml

```diff
@@ -77,9 +77,9 @@
 
         <!-- Versions of dependencies -->
         <asm.version>7.3.1</asm.version>
-        <hk2.version>2.6.1</hk2.version>
+        <hk2.version>3.0.0-M2</hk2.version>
         <jakarta-jms.version>3.0.0</jakarta-jms.version>
-        <hk2.plugin.version>2.6.1</hk2.plugin.version>
+        <hk2.plugin.version>3.0.0-M2</hk2.plugin.version>
```

Comment:
In GF: [nucleus/parent/pom.xml](https://github.com/eclipse-ee4j/glassfish/blob/d6f0dfa7adf823908d95bfc3e9490ec244d509ee/nucleus/parent/pom.xml#L125)